project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
 import sbt._
 
 object Dependencies {
-  private val akkaHttpCirceIntegrationV = "1.35.3"
-  private val akkaHttpV = "10.1.12" // scala-steward:off (CROM-6619)
-  private val akkaV = "2.5.31" // scala-steward:off (CROM-6637)
+  private val akkaHttpCirceIntegrationV = "1.37.0"
+  private val akkaHttpV = "10.1.14" // scala-steward:off (CROM-6619)
+  private val akkaV = "2.5.32" // scala-steward:off (CROM-6637)
   private val aliyunBcsV = "6.2.4"
-  private val aliyunCoreV = "4.5.20"
-  private val aliyunCrV = "4.1.1"
-  private val aliyunOssV = "3.11.3"
-  private val ammoniteOpsV = "2.3.8"
-  private val apacheCommonNetV = "3.8.0"
+  private val aliyunCoreV = "4.5.25"
+  private val aliyunCrV = "4.1.2"
+  private val aliyunOssV = "3.13.1"
+  private val ammoniteOpsV = "2.4.0"
   private val apacheHttpClientV = "4.5.13"
-  private val awsSdkV = "2.15.41" // scala-steward:off (CROM-6776)
+  private val awsSdkV = "2.17.29"
   private val betterFilesV = "3.9.1"
-  private val catsEffectV = "2.3.3"
-  private val catsV = "2.3.1"
+  /*
+  cats-effect, fs2, https4s, and sttp (also to v3) should all be upgraded at the same time to use cats-effect 3.x.
+   */
+  private val catsEffectV = "2.5.3" // scala-steward:off (CROM-6564)
+  private val catsV = "2.6.1"
```

Comment:
I don't know if this will affect us, but it's described as silently source-breaking: https://github.com/typelevel/cats/pull/3777 (from 2.4.1-hotfix). Can someone else check on this?

Reply:
I'm not sure if it will, but `Parallel` is used around CWL/WOM. So maybe? Although I've used a LOT of other parts of cats, I'm not super familiar with `Parallel`. I'm also not super familiar with Cromwell's language parsing, leaving me unsure if there would be negative tests to verify error accumulation works as expected. 🤔